ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/ReplaceLambdaWithInnerExpressionFix.fs

```diff
@@ -1,17 +1,34 @@
 ﻿namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
 
+open FSharp.Compiler
 open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
 open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Impl
 open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.ExtensionsAPI
 
 type ReplaceLambdaWithInnerExpressionFix(warning: LambdaCanBeReplacedWithInnerExpressionWarning) =
     inherit ReplaceWithInnerTreeNodeFixBase(warning.LambdaExpr, warning.ReplaceCandidate)
 
     let replaceCandidate = warning.ReplaceCandidate
 
+    let rec getRootFunctionExpr (app : IPrefixAppExpr) =
+        match app.FunctionExpression.IgnoreInnerParens() with
+        | :? IPrefixAppExpr as previousApp -> getRootFunctionExpr previousApp
+        | x -> x
+
     override x.Text =
         match replaceCandidate with
         | :? IReferenceExpr as ref when isSimpleQualifiedName ref ->
-            sprintf "Replace lambda with '%s'" ref.QualifiedName
-        | :? IPrefixAppExpr -> "Replace lambda with partial application"
+            $"Replace lambda with '{ref.QualifiedName}'"
+        | :? IPrefixAppExpr as app ->
+            match getRootFunctionExpr app with
+            | :? IReferenceExpr as ref ->
+                match ref.ShortName with
+                | name when name = SharedImplUtil.MISSING_DECLARATION_NAME ||
+                            PrettyNaming.IsOperatorName name ->
+                    $"Replace with '{ref.Reference.GetName()}' partial application"
```

Comment:
Why `Replace with '???' partial application` is a good text for the case of the missing name?

Reply:
The operator case shouldn't be any special now after [RIDER-41848](https://youtrack.jetbrains.com/issue/RIDER-41848) is fixed.